backend/otel/otel.go

```diff
@@ -300,9 +300,18 @@ func (o *Handler) HandleTrace(w http.ResponseWriter, r *http.Request) {
 					}
 				}
 
-				// skip logrus spans
-				if span.Name() == highlight.LogrusSpanName {
-					continue
+				// skip log spans
+				if span.Name() == highlight.LogrusSpanName || span.Name() == highlight.LogSpanName {
+					// skip unless workspace setting is on
+					settings, err := o.resolver.Store.GetAllWorkspaceSettingsByProject(ctx, fields.projectIDInt)
+					if err != nil {
+						log.WithContext(ctx).WithError(err).Error("failed to get workspace settings")
+						continue
+					}
+
+					if settings == nil || !settings.EnableLogTraceIngestion {
```

Comment:
nit: I would log the `settings == nil` case as an error since I don't think it's expected?